packages/ccd-js-gen/test/generate-test-bench.test.ts

```diff
@@ -0,0 +1,76 @@
+import * as Gen from '../src/lib.js';
+import { assertTypeChecks } from './testHelpers.js';
+
+const outDir = __dirname + '/lib';
+const outModuleFile = `${outDir}/test-bench`;
+const outContractFile = `${outDir}/test-bench_smart_contract_test_bench`;
+
+beforeAll(() => {
+    return Gen.generateContractClientsFromFile(
+        __dirname + '/resources/test-bench.wasm.v1',
```

Comment:
Can you please provide a header for this file explaining what the test is, what it depends on, and how to regenerate it.

Including how the wasm file was generated.